exercises/printUserInfo.js

```diff
@@ -27,16 +29,55 @@ console.log();
 console.log(`How's the weather in ${user['currentLocation']}?`);
 console.log();
 
-if (user['age'] >= 100) {
-  console.log(`Wow! You're ${user['age']} years old? Fewer than 0.2% reach that age.`);
+if (user['age'] >= 40) {
+  console.log(`Wow! You're ${user['age']} years old? Time for a crazy mid-life crisis!`);
   console.log();
 }
 
 console.log('Looks like you have great taste in music. Your favorite songs:');
 
-for (let i = 0; i < user['favoriteSongs'].length; i += 1) {
+printFavorites(user['favoriteSongs'])
+
+/** Given code for favortie songs list (for reference)
+  for (let i = 0; i < user['favoriteSongs'].length; i += 1) {
   let songPosition = i + 1;
   let songName = user['favoriteSongs'][i];
 
   console.log(`${songPosition}. ${songName}`);
 }
+*/
+
+console.log('I see that you eat ' + user['favoriteFood'] + '.')
+
+if ((user['favoriteFood'].toString()) === 'Hot Dogs') {
+  console.log(user['favoriteFood'] + ' are really processed!');
+} else if ((user['favoriteFood'].toString()) === 'Onigiri') {
+  console.log(user['favoriteFood'] + ' sounds delicious!');
+} else {
+  console.log(user['favoriteFood'] + ' is a good choice')
+}
+
+console.log()
+console.log('Your favorite movies are:')
+
+printFavorites(user['favoriteMovies'])
+
+/** Initial code for movie list (for reference)
+  for (let i = 0; i < user['favoriteMovies'].length; i += 1) {
+  let movieNumber = i + 1;
+  let movie = user['favoriteMovies'][i];
+
+  console.log(`${movieNumber}. ${movie}`);
+}
+console.log()
+*/
+
+function printFavorites (inputArray) {
+  for (let i = 0; i < inputArray.length; i += 1) {
+    let movieNumber = i + 1;
```

Comment:
I'd call this something like `itemNumber` since there's nothing about this function that has to do with movies in particular. You might use it with `user['favoriteSongs']`, too, for example.